src/common/Mutex.cc

```diff
@@ -92,6 +92,8 @@ Mutex::~Mutex() {
 void Mutex::Lock(bool no_lockdep) {
   int r;
 
+  r = 0;
```

Comment:
@tchaikov 
Arrgghh, you are right. The only when it is not initialize if 'goto out'  is called.
And then the assert is skipped.

But then this turns into the question why I get this assert in one of my wip's whiel running: unittest_librbd